src/components/webview/sourceControl.ts

```diff
@@ -20,6 +20,15 @@ export class SourceControlService {
 		private readonly __messageBus: MessageBus,
 	) {}
 
+	// @TODO
```

Comment:
do you know how we could get that?

Reply:
This should work https://stackoverflow.com/questions/28666357/how-to-get-default-git-branch
I will implement it in next PR